srcpkgs/gnome-shell/template

```diff
@@ -22,7 +22,7 @@ license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/GnomeShell"
 changelog="https://raw.githubusercontent.com/GNOME/gnome-shell/gnome-3-38/NEWS"
```

Comment:
the changelog should be updated to show GNOME 40 changes - https://raw.githubusercontent.com/GNOME/gnome-shell/gnome-40/NEWS

Reply:
this was not fixed, also please use their gitlab